docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -64,7 +57,18 @@ Only Applicable for Satisfaction Ratings object.
 **Table Name Field:** The name of the field that holds the table name. Must not be the name of any table column that
 will be read. Defaults to `tablename`.
 
-### Advanced
+### Connection
+
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
```

Comment:
Same comment about order of properties. Use Connection and Connection are usually first.

Reply:
For better visibility we have added the label Connection and order is correct according to that